src/react/features/home/home.feature

```diff
@@ -3,10 +3,12 @@ Feature: Home
   Background:
     Given I am at home
 
+  @pending
   Scenario: Rooms at home
     Given I have 2 rooms
     Then I see 2 rooms
 
+  @pending
```

Comment:
These are also "home-stuff" to be removed if we decide so...

Reply:
#23